defaults/main.yml

```diff
@@ -24,3 +24,5 @@ homebrew_cask_appdir: /Applications
 
 homebrew_use_brewfile: true
 homebrew_brewfile_dir: '~'
+
+homebrew_services: []
```

Comment:
This new variable should also be documented in the README, along with a link in that documentation block to https://github.com/Homebrew/homebrew-services so people can figure out how to use it.